drivers/include/sgp30.h

```diff
@@ -0,0 +1,257 @@
+ * =====
+ *
+ * This driver provides an interface for the Sensirion SGP30 Gas Sensor.
+ * The Datasheet can be found [here](https://www.mouser.com/pdfdocs/Sensirion_Gas_Sensors_SGP30_Datasheet_EN-1148053.pdf)
+ *
+ * After the sensor is powered up and starts measuring air quality for the
+ * first 15s @ref sgp30_read_measurements calls will return fixed values of
+ * of 400ppm CO2eq and 0ppb TVOC. Afterwards values should be read in regular
+ * intervals of 1s for best operation of the dynamic baseline compensationalgorithm.
+ *
+ * This above not done by default but if the `sgp30_strict` module is included then
+ * then if attempting to read before the is completed then -EAGAIN will be returned.
+ * After this periodic readings happen every 1s, and the last value read is
+ * returned when calling @ref sgp30_read_measurements. A timestamp is also added
+ * to @ref sgp30_data_t.
+ *
+ * The sensor features on-chip humidity compensation for the air quality
+ * signals. @ref sgp30_set_absolute_humidity can be used to change the
+ * absolute humidity value used. See [SGP30 driver integration] (https://files.seeedstudio.com/wiki/Grove-VOC_and_eCO2_Gas_Sensor-SGP30/res/Sensirion_Gas_Sensors_SGP30_Driver-Integration-Guide_HW_I2C.pdf)
+ * for more on this. The baseline values for the correction algorithm can
+ * also be tweaked with @ref sgp30_set_baseline. More on how to implement
+ * dynamic baseline compensation can be seen in at [SGP30 driver integration] (https://files.seeedstudio.com/wiki/Grove-VOC_and_eCO2_Gas_Sensor-SGP30/res/Sensirion_Gas_Sensors_SGP30_Driver-Integration-Guide_HW_I2C.pdf).
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ */
+
+#ifndef SGP30_H
+#define SGP30_H
+
+#include "periph/i2c.h"
+#include "ztimer.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief     Set of measured values
+ *
+ */
+typedef struct {
+    uint16_t tvoc;      /**< The last measurement of the IAQ-calculated Total
+                             Volatile Organic Compounds in ppb. */
+    uint16_t eco2;      /**< The last measurement of the IAQ-calculated
+                             equivalent CO2 in ppm */
+#ifdef MODULE_SGP30_STRICT
+    uint32_t timestamp; /**< Timestamp of last measured value */
+#endif
+} sgp30_data_t;
+
+/**
+ * @brief     Set of measured raw values
+ *
+ */
+typedef struct {
+    uint16_t raw_h2;        /**< raw H2 signal, only for testing purposes */
+    uint16_t raw_ethanol;   /**< raw Ethanol signal, only for testing purposes */
+} sgp30_raw_data_t;
+
+/**
+ * @brief   Device initialization parameters
+ */
+typedef struct {
+    i2c_t i2c_dev;              /**< I2C dev the sensor is connected to */
+} sgp30_params_t;
+
+/**
+ * @brief   Device descriptor for the driver
+ */
+typedef struct {
+    sgp30_params_t params;      /**< parameters of the sensor device */
+#ifdef MODULE_SGP30_STRICT
+    bool ready;                 /**< if initialization faze is over*/
```

Comment:
```suggestion
    bool ready;                 /**< if initialization phase is over*/
```

I think faze means about the opposite ;)